src/main/java/com/specialwarriors/conal/github_repo/exception/GithubRepoException.java

```diff
@@ -16,7 +16,8 @@ public enum GithubRepoException implements BaseException {
     EXCEED_GITHUBREPO_EMAIL(HttpStatus.BAD_REQUEST, "이메일은 5개까지 등록할 수 있습니다"),
     INVALID_GITHUBREPO_URL(HttpStatus.BAD_REQUEST, "잘못된 URL 입니다."),
     INVALID_GITHUBREPO_EMAIL(HttpStatus.BAD_REQUEST, "잘못된 이메일 입니다"),
```

Comment:
```suggestion
    EXCEED_GITHUB_REPO_EMAIL_LIMIT(HttpStatus.BAD_REQUEST, "이메일은 5개까지 등록할 수 있습니다"),
    INVALID_GITHUB_REPO_URL(HttpStatus.BAD_REQUEST, "잘못된 URL 입니다."),
    INVALID_GITHUB_REPO_EMAIL(HttpStatus.BAD_REQUEST, "잘못된 이메일 입니다"),
```
- 코드 컨벤션에 예외명이 어긋나는 듯 보이는데 수정 부탁드립니다
- 예외 상황을 좀 더 명확히 표현할 수 있도록 예외명을 변경하는 것이 어떨까요?

Reply:
컨벤션에 맞도록 수정하겠습니다!